torch/amp/autocast_mode.py

```diff
@@ -170,7 +170,9 @@ def forward(self, x):
     (see :ref:`Working with Multiple GPUs<amp-multigpu>`).
 
     Args:
-        device_type(str, required):  Whether to use 'cuda' or 'cpu' device
+        device_type(str, required):  Whether to use 'cuda' or 'cpu' device.
```

Comment:
Actually, there are more than `cuda` and `cpu` are supported for now, refer to this: https://github.com/pytorch/pytorch/blob/95621b3c2e8bc0f1d7f88fb0bcb3e3e56a0ff961/torch/amp/autocast_mode.py#L193-L218.

Reply:
I noticed that the error message on L218 in the snippet you posted is also out-dated.